app/controllers/editor_controller.rb

```diff
@@ -17,9 +18,27 @@ def post
       params[:body] = node.body if node
     end
     redirect_to "/questions/new?#{request.env['QUERY_STRING']}" if params[:tags] && params[:tags].include?("question:")
+    return "/editor/post"
```

Comment:
Getting closer! I see the error:

`ActionView::MissingTemplate: Missing template editor/legacy, application/legacy with {:locale=>[:en], :formats=>[:html], :handlers=>[:erb, :builder, :coffee, :jbuilder]}. Searched in:`

I think you need to add a `render template: 'editor/post'` as the final line in this method, and `return "/editor/post"` doesn't seem necessary -- unless I'm missing something?

Thanks!